main.py

```
@@ -1,8 +1,7 @@
 #!/usr/bin/env python3
 import sys
-
 sys.dont_write_bytecode = True
-import sys
+from utils.operator.cleanup import cleanup
```

Ruff (0.8.2)

`utils.operator.cleanup.cleanup` imported but unused

Remove unused import: `utils.operator.cleanup.cleanup`

(F401)